django_freeradius/tests/test_api.py

```diff
@@ -52,3 +53,140 @@ def test_postauth_400(self):
         response = self.client.post(reverse('freeradius:postauth'), {})
         self.assertEqual(RadiusPostAuth.objects.all().count(), 0)
         self.assertEqual(response.status_code, 400)
+
+
+    def test_accounting_start(self):
+        RadiusAccounting.objects.all().count()
```

Comment:
remove this line because the count is not being used in anyway and is not useful